### PR TITLE
[DotNetCore] Added ASP.NET Core specific RunConfiguration settings(AppURL and LaunchBrowser)

### PR DESCRIPTION
**Pull request by DavidKarlas**
Pretty happy with end result:
image
Application starts on correct url and browsers launches right after server starts listening(same as VS.net does it, by trying to connect to web server address).

### CODE REVIEW

File: main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/DotNetCoreRunConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+// THE SOFTWARE.
+using System;
+using System.Net;
+using MonoDevelop.Core.Serialization;
+using MonoDevelop.Projects;
+using System.Net.Sockets;
+
+namespace MonoDevelop.DotNetCore
+{
+	public class DotNetCoreRunConfiguration : AssemblyRunConfiguration
```
Comment (mrward) [inline main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/DotNetCoreRunConfiguration.cs]: Can we make this non-public?

**Comment by slluis** [comment]
What is the checkbox in Launch Url for?

**Comment by DavidKarlas** [comment]
To launch or not to launch browser... I agreed it looks akward, but I wanted to keep it similar to VS:
image

**Comment by slluis** [comment]
And what's the difference between Launch Url and App Url?

**Comment by DavidKarlas** [comment]
App URL is URL on which server will listen.(http://localhost:8888)
Launch URL is where browser will be pointed to.(http://localhost:8888/website/i/want/to/open/without/navigating/over/20/clicks) Launch URL can also be relative, like just "index.html") which means combine App URL... which results in http://localhost:8888/index.html

**Comment by DavidKarlas** [comment]
It's also possible that you are developing micro web service and you want to launch in browser totally different web address that is calling your micro web service, which makes sense to have totally different absolute(host+port) address

**Comment by slluis** [comment]
Now, let's redesign that panel so that users don't have to ask the above questions. /cc @vancura @hbons

**Comment by slluis** [comment]
Also, why is it necessary to specify the full App Url? shouldn't the port number be enough?

**Comment by DavidKarlas** [comment]
So user can specify on which domain/ip to listen in case of remote deployment/debugging... I guess we could change logic to append http://localhost in case only port is specified...
But for now I would prefer to keep maximum compatibility with VS to not confuse users...

**Comment by hbons** [comment]
I'll have a look at this.

**Pull request merged**